src/renderer/qss/js/qssSettingButtons.js

```diff
@@ -49,14 +48,26 @@
      * on whether the changes to the setting can be undone and whether the new value of
      * the setting is the same as its default one.
      * @param {Component} that - The `gpii.qss.settingButtonPresenter` instance.
-     * @param {Object} setting - The setting object corresponding to this QSS button.
-     * @param {Any} value - The new value of the `setting`.
+     * @param {Object} setting - The setting object corresponding to this QSS button
```

Comment:
Since this structure has now become significantly complex, please can you add a @typedef JSdoc entry for it explaining all the structures it can have, and update any references in function arguments